src/languages/graphql.js

```diff
@@ -15,16 +15,16 @@ export default function(hljs) {
     keywords: {
       keyword: [
         "query",
-        "mutation",
-        "subscription|10",
+        "mutation|10",
```

Comment:
```suggestion
        "mutation",
```

If you're having relevance issues, just throw those to me at the end to solve.  The correct fix isn't to boost random words that we include in `default`...

Reply:
ok, but the tests are going to fail

Reply:
Let me deal with the tests...

Reply:
`graphql` *might* be reasonable to consider for the default bundle even, and if so it needs careful review in regard to relevance vs all the others, etc...